readinglist/views/article.py

```diff
@@ -67,6 +67,7 @@ class Options:
 @crud()
 class Article(BaseResource):
     mapping = ArticleSchema()
+    deleted_mark = ('status', 2)
```

Comment:
What's wrong with ``status : 2`` ?

We couldn't patch on ``deleted`` either!

You remind me of something : we may want to filter on status=2 :) I'll add a test to check that it works :)